brokers/vscode/cmd/main.go

```diff
@@ -0,0 +1,36 @@
+//
+// Copyright (c) 2012-2018 Red Hat, Inc.
+// This program and the accompanying materials are made
+// available under the terms of the Eclipse Public License 2.0
+// which is available at https://www.eclipse.org/legal/epl-2.0/
+//
+// SPDX-License-Identifier: EPL-2.0
+//
+// Contributors:
+//   Red Hat, Inc. - initial API and implementation
+//
+
+package main
```

Comment:
In other brokers you did the following structure:
```
/brokers
    /che-plugin-broker
      /broker
          broker.go
      main.go
    /theia-plugin-broker
      /broker
          broker.go
      main
```
But for vscode you have different one:
/brokers
    /vscode
      /cmd
          main.go
      broker.go
```
I think it would be better to have the same folders structure for all plugin brokers

Reply:
done